docs/index.html

```diff
@@ -183,7 +186,15 @@ <h1 style="text-align: center;">GeoRust</h1>
               <td class="px-4 py-3">STAC</td>
               <td class="px-4 py-3 font-normal">Work with SpatioTemporal Asset Catalogs (STAC)</td>
               <td class="px-4 py-3 text-indigo-900"><a href="https://github.com/gadomski/stac-rs">GitHub</a></td>
-              <td class="px-4 py-3 text-indigo-900"><a href="https://github.com/gadomski/stac-rs">crates.io</a></td>
+              <td class="px-4 py-3 text-indigo-900"><a href="https://crates.io/crates/stac">crates.io</a></td>
+            </tr>
+            <tr>
+              <td class="px-4 py-3">PgSTAC</td>
```

Comment:
Please maintain the alphabetical ordering.

Reply:
Sorry, the entries weren't alphabetical to begin with ... would you like me to correct the whole section?

Reply:
I see the alphabetical ordering ship sailed a few commits ago... I guess it's not worth stressing over.